examples/evaluate/06-TBROM_input_field.py

```diff
@@ -0,0 +1,195 @@
+    )
+    results.plot(
+        x=x_ind,
+        y=y2_ind,
+        ax=axes0,
+        ls="-.",
+        label="{}".format("Maximum Deformation (output field reconstruction on Group_2)"),
+    )
+
+    axes0.set_title("T-junction deformation response")
+    axes0.set_xlabel(results.columns[x_ind] + " [Pa]")
+    axes0.set_ylabel("Deformation [m]")
+
+    # Show plot
+    plt.show()
+
+
+def norm_vector_field(field: list):
+    """Compute the norm of a vector field."""
+
+    norm = []
+    for i in range(0, int(len(field) / 3)):
+        x = field[i * 3]
+        y = field[i * 3 + 1]
+        z = field[i * 3 + 2]
+        norm.append(math.sqrt(x * x + y * y + z * z))
+    return norm
+
+
+###############################################################################
+# Load the twin runtime and generate temperature results
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Load the twin runtime and generate temperature results from the TBROM.
+
+print("Loading model: {}".format(twin_file))
+twin_model = TwinModel(twin_file)
+
+###############################################################################
+# Evaluate the twin with different input values and collect corresponding outputs
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Because the twin is based on a static model, two options can be considered:
+#
+# - Set the initial input value to evaluate and run the initialization function (current approach).
+# - Create an input dataframe considering all input values to evaluate and run the batch function
+#   to evaluate. In this case, to execute the transient simulation, a time dimension must be
+#   arbitrarily defined.
+
+results = []
+input_name_all = list(twin_model.inputs.keys())
+output_name_all = list(twin_model.outputs.keys())
+
+# remove the TBROM related pins from the Twin's list of inputs and outputs
+input_name_without_mcs = []
+for i in input_name_all:
+    if "mode" not in i:
+        input_name_without_mcs.append(i)
+print(f"Twin physical inputs : {input_name_without_mcs}")
+output_name_without_mcs = []
+for i in output_name_all:
+    if "mode" not in i:
+        output_name_without_mcs.append(i)
+print(f"Twin physical outputs : {output_name_without_mcs}")
+
+# collect the TBROM and input field related information, the Twin must be initialized first
+twin_model.initialize_evaluation()
+print(f"TBROMs part of the Twin : {twin_model.tbrom_names}")
+romname = twin_model.tbrom_names[0]
+print(f"Input fields associated to the TBROM {romname} : {twin_model.get_rom_inputfieldsnames(romname)}")
+fieldname = twin_model.get_rom_inputfieldsnames(romname)[0]
+print(f"Named selections associated to the TBROM {romname} : {twin_model.get_rom_nslist(romname)}")
+ns = twin_model.get_rom_nslist(romname)[1]
+
+input_name = input_name_without_mcs[0]
+for i in range(0, len(rom_inputs)):
+    # Initialize twin with input values and collect output value
```

Comment:
```suggestion
    #initialize twin with input values and collect output value
```
While you can use initial capital or lowercase, the goal is to be consistent within an example. Within the code, other comments start in lowercase.